### ISSUE TITLE
Manually Install Plugin Failed

### ISSUE THREAD

**Issue by xianlin** [huaweicloud/packer-plugin-huaweicloud#111]
How to install the latest release? seems auto install doesn't work, it keeps showing version "0.4.0-rc1" instead of v1.2.0?

```
packer plugins install -path ~/Downloads/packer-plugin-huaweicloud_v1.2.0_x5.0_linux_386 "github.com/huaweicloud/huaweicloud"
Error: Invalid version

Packer can only install plugin releases with this command (ex: 1.0.0) or
development pre-releases (ex: 1.0.0-dev), the binary's reported version is
"0.4.0-rc1"
```

how to install the latest version using `packer init` ?

```
packer {
  required_plugins {
    huaweicloud = {
      version = ">= 1.0.0"
      source  = "github.com/huaweicloud/huaweicloud"
    }
  }
}



packer init .
Failed getting the "github.com/huaweicloud/huaweicloud" plugin:
8 errors occurred:
	* Continuing to next available version: binary reported version ("0.4.0-rc1") is different from the expected "1.2.0", skipping
	* Continuing to next available version: binary reported version ("0.4.0-rc1") is different from the expected "1.1.0", skipping
	* Continuing to next available version: binary reported version ("0.4.0-rc1") is different from the expected "1.0.4", skipping
	* Continuing to next available version: binary reported version ("0.4.0-rc1") is different from the expected "1.0.3", skipping
	* Continuing to next available version: binary reported version ("0.4.0-rc1") is different from the expected "1.0.2", skipping
	* Continuing to next available version: binary reported version ("0.4.0-rc1") is different from the expected "1.0.1", skipping
	* Continuing to next available version: binary reported version ("0.4.0-rc1") is different from the expected "1.0.0", skipping
	* could not install any compatible version of plugin "github.com/huaweicloud/huaweicloud"

```

**Comment by Zippo-Wang** [comment]
hello Mr @xianlin , we have carefully reviewed your message. Packer official has made some breaking change since 1.11.x version, see [packer changelog](https://github.com/hashicorp/packer/blob/main/CHANGELOG.md).
Here are two advice:
1. You can use 1.10.x or earlier version to install lasted packer-plugin-huaweicloud
```hcl
# init.pkr.hcl
packer {
  required_plugins {
    huaweicloud = {
      version = ">= 1.0.0"
      source  = "github.com/huaweicloud/huaweicloud"
    }
  }
}
```

then use `packer init init.pkr.hcl` to install lastet version

2. You can use 1.11.x continuelly, and manually get lastet version from github [packer huaweicloud release](https://github.com/huaweicloud/packer-plugin-huaweicloud/releases). Then put it to `/root/.packer.d/plugins`, such as
![image](https://github.com/user-attachments/assets/295a1530-22af-4d37-bb0f-fc08c123c2a2)
 
then, you do not need to use `packer init` to install plugin
finally, use `packer build test.pkr.hcl` to create image
![image](https://github.com/user-attachments/assets/494ebb27-58f6-4ea2-98b5-f952f07da5b0)

**Comment by xianlin** [comment]
@Zippo-Wang , thank you and for 2nd option - manually install the plugin, what should I put in my packer block? 

I must put something there or it will report error.

```
packer {
  required_plugins {
    huaweicloud = {
      version = ">= 0.4.0"
      source  = "github.com/huaweicloud/huaweicloud"
    }
  }
}

```

**Comment by Zippo-Wang** [comment]
Due to packer official has made breaking change and SDK incompatible, it maybe cause some unknown trouble.
We are one of plugin of packer. I recommend you to use the first method if you want to use lastet version(packer-huaweilcloud 1.2.0). So that you can use `packer init` to continue your operation.

**Comment by chengxiangdong** [comment]
This issue has been fixed in version `v1.2.1`, please download and use it.
